setup.cfg

```diff
@@ -75,6 +75,11 @@ apache.hive =
     impyla
 microsoft.azure =
     apache-airflow-providers-microsoft-azure
+openlineage =
+    openlineage-airflow==0.6.2
```

Comment:
We should check what happens if `openlineage-airflow` is not installed, does it give an error that the library needs to be installed even if the user does not want to use lineage?

This is not a problem for ou r cloud users, but for OSS users it will be